.eslintrc

```
@@ -0,0 +1,89 @@
+{
+  "extends": ["plugin:@typescript-eslint/recommended", "prettier", "plugin:prettier/recommended"],
```

Add installed react plugin

```suggestion
  "extends": ["plugin:@typescript-eslint/recommended", "prettier", "plugin:prettier/recommended", "plugin:react/recommended"],
```